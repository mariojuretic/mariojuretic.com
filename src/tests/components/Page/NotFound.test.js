import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../../components/Page/NotFound';

test('should render PageNotFound correctly', () => {
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
});
