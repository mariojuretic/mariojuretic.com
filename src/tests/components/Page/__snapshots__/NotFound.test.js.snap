// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound correctly 1`] = `
<div>
  404 - Page Not Found
</div>
`;
